liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -22,9 +22,13 @@
 #include <liblangutil/Scanner.h>
 #include <liblangutil/Exceptions.h>
 #include <libsolutil/UTF8.h>
+#include <libsolutil/hyperlink.h>
```

Comment:
This is missing in the commit. Probably explains the CI.

Reply:
Oops!